project/plugins.sbt

```diff
@@ -5,13 +5,12 @@ resolvers ++= Seq(
 )
 
 addSbtPlugin("com.eed3si9n"       % "sbt-unidoc"      % "0.3.3")
-addSbtPlugin("com.fortysevendeg"  % "sbt-microsites"  % "0.3.0")
+addSbtPlugin("com.fortysevendeg"  % "sbt-microsites"  % "0.3.3")
 addSbtPlugin("com.github.gseitz"  % "sbt-release"     % "1.0.0")
 addSbtPlugin("com.jsuereth"       % "sbt-pgp"         % "1.0.0")
 addSbtPlugin("com.typesafe"       % "sbt-mima-plugin" % "0.1.11")
 addSbtPlugin("com.typesafe.sbt"   % "sbt-ghpages"     % "0.5.3")
 addSbtPlugin("com.typesafe.sbt"   % "sbt-scalariform" % "1.3.0")
 addSbtPlugin("org.scoverage"      % "sbt-scoverage"   % "1.3.5")
-addSbtPlugin("org.tpolecat"       % "tut-plugin"      % "0.4.4")
```

Comment:
we don't need this explicit dependency. sbt-microsites upgrades will bring it along for the ride.